week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/model/index.ts

```diff
@@ -0,0 +1,14 @@
+export type Currency = 'PLN' | 'USD'
+export type Unit = 'kg' | 'g'
+
+export interface Price {
+  value: number;
+  currency: Currency;
+}
+
+export interface Item {
+  name: string;
+  amount: number;
+  unit: Unit;
+  price: Price;
+}
```

Comment:
Jedyne co można poprawić w mojej opinii - to wydobycie tych modeli do "jawnych" plików:

`model/Price.ts`
`model/Item.ts`

na tej zasadzie `Currency` moze mieszkać z `Price` a `Unit` z `Item` na razie.